packages/beacon-node/src/api/impl/node/index.ts

```diff
@@ -75,7 +75,7 @@ export function getNodeApi(
 
       let healthStatus: number;
 
-      if (sync.getSyncStatus().isSyncing) {
+      if (sync.getSyncStatus().isSyncing ||  (!sync.getSyncStatus().isSyncing  && syncingStatus.getSyncStatus().isOptimistic)) {
```

Comment:
should avoid calling `.getSyncStatus()` multiple times, the condition is also either node is syncing **or** EL is optimistic

see https://github.com/ethereum/beacon-APIs/pull/442/files#diff-cc32cc26efdb358fa3338a84d59b700df00098f61d1ecbdacd5545ed0bccb31aR20